frontend/src/scenes/settings/project/PersonsOnEvents.tsx

```diff
@@ -28,7 +28,7 @@ const poeOptions: LemonRadioOption<PoEMode>[] = [
             <>
                 <div>Use person properties from the time of the event.</div>
                 <div className="text-muted">
-                    Faster queries. If person property is updated, then query results on past data won't change.
+                    Fast queries. If person property is updated, then query results on past data won't change.
```

Comment:
```suggestion
                    Faster queries. If person property is updated, then query results on past data won't change.
```

Reply:
because they aren't all that fast sometimes 😅  it's fine to have 'fastest' and 'faster'